app/src/main/java/ar/com/wolox/android/cookbook/googleLogin/GoogleLoginActivity.kt

```diff
@@ -0,0 +1,13 @@
+package ar.com.wolox.android.cookbook.googleLogin
+
+import ar.com.wolox.android.cookbook.R
+import ar.com.wolox.wolmo.core.activity.WolmoActivity
+
+class GoogleLoginActivity : WolmoActivity() {
```

Comment:
Mmm, this name doesn't indicate it's a recipe. I'd rename this `Activity` and its child `Fragment` to:

* `GoogleLoginRecipeActivity`
* `GoogleLoginRecipeFragment`